recipes/spatialpandas/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "spatialpandas" %}
+{% set version = "0.3.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0718171c3230f01b1c4ed89403044a41aedf305e8934a635ff368ffe5b52f045
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - param
```

Comment:
Is param really needed at package build time?

Reply:
I am not the author of the project, they import it in their setup.py file for getting a formatted version number I think (see last failed CI build on this PR and their setup.py). Likely something that could be fixed in that project.

Reply:
I see OK